server/src-lib/Hasura/Server/Init.hs

```diff
@@ -532,6 +532,14 @@ parseRawConnInfo =
                   metavar "<DBNAME>" <>
                   help "Database name to connect to"
                 )
+
+    options = optional $
+      strOption ( long "options" <>
```

Comment:
I would probably call this `pg-connection-options` or something along those lines, `options` is too generic.

Reply:
Done.